tests/test_templater.py

```diff
@@ -0,0 +1,89 @@
+#pylint: disable=unused-variable
```

Comment:
From a test robustness perspective, I might make sure there are tests for:

The `file_exists` function itself - properly throws error, returns the path and the `parse_args()` function itself and that the correct environment variables (file or env) are used in `set_template()`